science/rtl_433/Portfile

```diff
@@ -5,26 +5,33 @@ PortGroup           cmake 1.1
 PortGroup           github 1.0
 
 epoch               1
-github.setup        merbanan rtl_433 21.12
-# Change github.tarball_from to 'releases' or 'archive' next update
-github.tarball_from tarball
+github.setup        merbanan rtl_433 25.02
+github.tarball_from archive
 revision            0
 
 categories          science comms
 license             GPL-2
-maintainers         {@ducksauz duksta.org:john} openmaintainer
+maintainers         nomaintainer
```

Comment:
where did the maintainer state that he doesn't want/can maintain the port anymore?

Reply:
Hi! Retiring maintainer here. Please go ahead and remove me.